pkg/analytics/reporter.go

```diff
@@ -71,12 +78,26 @@ func NewReporter(config Config, kvConfig kv.Config, objectClient client.ObjectCl
 	if !config.Enabled {
 		return nil, nil
 	}
+
+	originalDefaultTransport := http.DefaultTransport.(*http.Transport)
+	tr := originalDefaultTransport.Clone()
+	if config.TLSConfig.CertPath != "" || config.TLSConfig.KeyPath != "" {
```

Comment:
I don't see the proxy to be configured. `Proxy` is part of the `Transport` of the `http.Client`
https://pkg.go.dev/net/http#DefaultTransport

Reply:
my bad, that was a leftover of my first approach. We don't have to use the proxy_url anymore. I'm pushing a commit removing it.
Uses can configure a proxy through the HTTPS_PROXY environment variable.

Reply:
I stepped back from not using `proxy_url` and added it back. Otherwise all requests that rely on the default environment proxy would be afflicted by it. I added a unitary test covering its usage.